### DISCUSSION TITLE
Windows Arm/Arm64

### DISCUSSION THREAD

**Discussion by Myqin01** [OpenRCT2/OpenRCT2#16749] [Feature requests]
New to Github, so sorry if this is the wrong place to submit this request. Since this is an open source project, would it be too difficult to provide an Arm/Arm64 Windows binary? Or at least instructions to compile the source to Arm/Arm64? I know Arm on Windows is still a little niche, but this would be a great time killer for people that made the plunge like myself. It does _work_ in x86/x86_64 emulation, but you can really feel the CPU chug as soon as any sort of physics is calculated. For instance, on my Surface Pro X SQ-1, it will idle around 80 fps - until any sort of car/train moves. Then it drops to 10-20fps.

Regardless if the OpenRCT2 team can/wants to meet this feature request, you guys are doing incredible work on this project. Much love<3

**Comment by IntelOrca** [comment]
@janisozaur has been working towards getting Windows ARM builds working. There was an issue here: https://github.com/OpenRCT2/OpenRCT2/issues/6682, although this refers to UWP which I am not sure is necessary for your case.

There is already an ARM dependencies release, so possibly all that remains is setting up the CI to build an ARM binary. Generation of g2.dat will need to be copied from another CI job, or created with a standalone tool (https://github.com/OpenRCT2/OpenRCT2/pull/16107).

It looks like there is already a trial build you can try here:
https://github.com/janisozaur/arm64-binaries/tree/master/openrct2

**Reply by Myqin01** [reply]
Thanks a bunch!

**Reply by janisozaur** [reply]
@myqin01 can you say how well/badly it performs, especially compared to x86/amd64 build? I'm working on updating that build as well, need some more time to finish it up.

**Reply by janisozaur** [reply]
There is an updated build available through https://github.com/OpenRCT2/OpenRCT2/pull/16887

**Reply by Myqin01** [reply]
> @Myqin01 can you say how well/badly it performs, especially compared to x86/amd64 build? I'm working on updating that build as well, need some more time to finish it up.

I didnt see the email about the mention, but thank you for responding. Performance is hard to say exactly. Under x86/x64 emulation, if not much is going on, performance is ok. Like, the second the game has to calculate any sort of physics, performance falls off a cliff. I'm using a Surface Pro X SQ1 with 8GB of RAM. If I just start a park, and the only thing happening is people walking around, about 80FPS. The second that a ride begins moving, down to 20fps.



> There is an updated build available through #16887

Nice! I'll take a peek to see if I can be useful as a tester.

**Comment by J-PIE-314** [comment]
Windows ARM64 builds are now available. You can download a release version here: https://github.com/OpenRCT2/OpenRCT2/releases
